pkg/disk/utils.go

```diff
@@ -0,0 +1,63 @@
+package disk
```

Comment:
Please rename this file to disk.go

Reply:
Why do you think it should be renamed to `disk.go`? This is a collection of utils relevant to the disk export process.

Reply:
The name utils does not say anything about what is inside of that file. 
AFAIR Kubevirt is trying to get rid of these files called utils.